src/screens/LoginScreen.js

```diff
@@ -1,7 +1,7 @@
 // 3rd part libraries - core
 import {AppLoading} from 'expo';
 import React, {Component} from 'react';
-import {View, Text} from 'react-native';
+import {View, Text, ImageBackground} from 'react-native';
```

Comment:
Oh awesome! We need this for the `AuthScreen` because right now after you log in with facebook and it's doing the Firebase login in the background, you just sit on this blank screen. I'll make an issue and maybe you can take a crack at fixing that bug with an ImageBackground styling?

Reply:
Just created it, GH-38